rotkehlchen/inquirer.py

```diff
@@ -428,8 +429,12 @@ def _query_oracle_instances(
         for oracle, oracle_instance in zip(oracles, oracle_instances):
             if (
                 isinstance(oracle_instance, CurrentPriceOracleInterface) and
-                oracle_instance.rate_limited_in_last() is True
+                oracle_instance.rate_limited_in_last(DEFAULT_RATE_LIMIT_WAITING_TIME) is True
             ):
+                log.warning(
```

Comment:
Do we need to add this warning? Won't it spam the user A LOT? This will happen very often.

Reply:
is in the logs, not in the msg aggregator

Reply:
That changes very little. Spamming the logs is equally bad.

1. Consumes resources
2. Makes logs very hard to read

 If you want to log this Isn't it better to log at the time of rate limiting and for how long?
The way you have it right now what will be happening is that in each iteration of the for loop for each oracle query you will be logging this warning so long as the oracle instance is rate limited.

Reply:
okey, got it